### PR TITLE
DCS-33 Adding elite2api dependency on docker-compose

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
+
+  oauth-server:
+    image: mojdigitalstudio/nomis-oauth2-server:latest
     networks:
       - hmpps
+    container_name: oauth-server
+    ports:
+      - "9090:8080"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost8080/auth/health"]
```
Comment (petergphillips) [inline docker-compose.yml]: missing colon

File: docker-compose.yml [hunk 2]
```diff
@@ -1,25 +1,31 @@
...
     environment:
-    - PORT=3000
-    - DB_USER=use-of-force
-    - DB_PASS=use-of-force
-    - DB_SERVER=use-of-force-db
-    - DB_NAME=use-of-force
-    - DB_SSL_ENABLED=no
-    - NOMIS_AUTH_URL=http://nomis-oauth2-server:8080/auth
-    - NOMIS_AUTH_EXTERNAL_URL=http://localhost:8080/auth
+      - SERVER_PORT=8080
```
Comment (petergphillips) [inline docker-compose.yml]: don't need server port

File: docker-compose.yml [hunk 3]
```diff
@@ -1,25 +1,31 @@
...
+    image: mojdigitalstudio/nomis-oauth2-server:latest
     networks:
       - hmpps
+    container_name: oauth-server
+    ports:
+      - "9090:8080"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost8080/auth/health"]
+    environment:
+      - SERVER_PORT=8080
```
Comment (petergphillips) [inline docker-compose.yml]: don't need server port

Comment (andrewrlee) [reply docker-compose.yml]: removed

File: docker-compose.yml (hunk 2 above)
Comment (andrewrlee) [reply docker-compose.yml]: removed

File: docker-compose.yml (hunk 1 above)
Comment (andrewrlee) [reply docker-compose.yml]: fixed

**Pull request merged**